<launch>
    

    <node name="kittiHelper" pkg="LiPMatch" type="kittiHelper" output="screen"> 
        <param name="dataset_folder" type="string" value="/home/jjwen/data/KITTI/odometry/" />
        <param name="sequence_number" type="string" value="00" />
        <param name="to_bag" type="bool" value="false" />
        <param name="output_bag_file" type="string" value="/home/jjwen/data/KITTI/odometry/00/kitti.bag" /> <!-- replace with your output folder -->
        <param name="publish_delay" type="int" value="1" />

    </node>
</launch>
